build.sh

```diff
@@ -18,7 +18,7 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean libcugraph cugraph docs -v -g -n --show_depr_warn -h --help"
+VALIDARGS="clean libcugraph cugraph docs -v -g -n --allgpuarch --show_depr_warn -h --help"
 HELP="$0 [<target> ...] [<flag> ...]
```

Comment:
While you're updating this, can you also update the part that describes the default actions (ie. "default action (no args) is to...") since it's out of date and doesn't mention doc builds.

Reply:
Okay, what should that say? 

" default action (no args) is to build and install 'libcugraph' then 'cugraph' targets *_and then documentation_*"

That okay?

Reply:
looks better, thanks.